src/transformers/configuration_bert_for_seq_generation.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
+# Copyright 2020 The Google AI Language Team Authors and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""  BertForSeqGeneration model configuration """
+
+from .configuration_utils import PretrainedConfig
+
+
+class BertForSeqGenerationConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a :class:`~transformers.BertForSeqGenerationConfig`.
+    It is used to instantiate an BertForSeqGenerationConfig model according to the specified arguments, defining the model
+    architecture.
+
+    Configuration objects inherit from  :class:`~transformers.PretrainedConfig` and can be used
+    to control the model outputs. Read the documentation from  :class:`~transformers.PretrainedConfig`
+    for more information.
+
+
+    Args:
+        vocab_size (:obj:`int`, optional, defaults to 50358):
```

Comment:
Can we have all optional -> `optional`.

I know I haven't cleaned up all config/model/tokenizer docstrings yet but it'd be great to use the same format as in the cleaned one :-)

Reply:
yes! Did a find and replace - should be fine :-)